Tests/PubNubContractTest/Steps/Files/PubNubFilesContractTestSteps.swift

```diff
@@ -156,5 +156,75 @@ public class PubNubFilesContractTestSteps: PubNubContractTestCase {
 
       self.wait(for: [sendFileExpect], timeout: 60.0)
     }
+    
+    When("^I send a file with '(.+)' space id and 'this-is-really-long-message-type-to-be-used-with-publish' message type$") { args, _ in
+      let spaceId = args?.first ?? String()
+      let messageType = "this-is-really-long-message-type-to-be-used-with-publish"
+      
+      let sendFileExpect = self.expectation(description: "Send file Response")
+
+      guard let data = "test file data".data(using: .utf8) else {
+        XCTAssert(false, "Unable prepare file data")
+        return
+      }
+      
+      self.client.fileURLSession = URLSession(
+        configuration: .default,
+        delegate: self.client.fileSessionManager,
+        delegateQueue: .main
+      )
+
+      let publishFileRequest = PubNub.PublishFileRequest(
+        messageType: .user(type: messageType),
+        spaceId: PubNubSpaceId(spaceId)
+      )
+      
+      self.client.send(.data(data, contentType: nil), channel: "test", remoteFilename: "name.txt", publishRequest: publishFileRequest) { result in
+        switch result {
+        case let .success(sendResults):
+          self.handleResult(result: sendResults)
+        case let .failure(error):
+          self.handleResult(result: error)
+        }
+        sendFileExpect.fulfill()
+      }
+
+      self.wait(for: [sendFileExpect], timeout: 60.0)
+    }
+    
+    When("^I send a file with '(.+)' space id and 'ts' message type$") { args, _ in
```

Comment:
@jguz-pubnub because matched list can dynamically provide you with values, these step looks like redundant if suggested changes will be done.